docs/releases/4.1.txt

```diff
@@ -355,6 +355,10 @@ Tests
   :meth:`~.SimpleTestCase.assertFormsetError` now support passing a
   form/formset object directly.
 
+* The :class:`.DiscoverRunner` now supports running tests in parallel on
+  systems, such as macOS and Windows, where the ``spawn``
+  :mod:`multiprocessing` start method is used.
```

Comment:
A little word smithing...

```suggestion
* The :class:`.DiscoverRunner` now supports running tests in parallel on
  macOS, Windows, and any other systems where the default
  :mod:`multiprocessing` start method is ``spawn``.
```

Could also go as the first point in the section? Much bigger news than the others!

Reply:
Done. Thanks! 👍